### ISSUE TITLE
Warnings in HA logs:  Detected code that accesses the database without the database executor;

### ISSUE THREAD

**Issue by VictorvandenHoven** [uvejota/homeassistant-edata#91]
Not sure if it is related to the edata-addon, but after using the edata-connector I got the following warnings in my HA logs:

```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:74
First occurred: 12:01:17 (32 occurrences)
Last logged: 15:32:47

Detected code that accesses the database without the database executor; Use homeassistant.components.recorder.get_instance(hass).async_add_executor_job() for faster database operations. Please report this issue.
```

The edata-connector works great by the way, thank you very much.

**Comment by sergidelatorre** [comment]
+1 Same issue

**Comment by dragonjon** [comment]
+1 Same issue
#87 
@uvejota

**Comment by dragonjon** [comment]
[edata.zip](https://github.com/uvejota/homeassistant-edata/files/8645024/edata.zip)
my edata files, resolved errors
modified from [2022.01.0](https://github.com/uvejota/homeassistant-edata/releases/tag/2022.01.0)

**Comment by dragonjon** [comment]
Duplicate of #87

**Comment by dave-code-ruiz** [comment]
+1 same issue

**Comment by Siroguh** [comment]
@dragonjon solved all my problems. Thanks!

**Comment by dave-code-ruiz** [comment]
> [edata.zip](https://github.com/uvejota/homeassistant-edata/files/8645024/edata.zip) my edata files, resolved errors modified from [2022.01.0](https://github.com/uvejota/homeassistant-edata/releases/tag/2022.01.0)

Solved my problems too

**Comment by uvejota** [comment]
Corregido en 2022.01.1

Gracias @dragonjon :)
